scripts/qa_crowbarsetup.sh

```diff
@@ -1248,7 +1249,7 @@ function onadmin_add_cloud_repo()
         echo "$clouddescription"
         echo
     fi
-    ) >> /etc/motd
+    ) > /etc/motd
```

Comment:
Yes. Because with combined SP1+SP2 setup, I have 2 lines in motd, and I do not like it. AFAIK motd is not used for anything else